src/os/bluestore/BlueFS.cc

```diff
@@ -18,6 +18,19 @@
 #undef dout_prefix
 #define dout_prefix *_dout << "bluefs "
 using TOPNSPC::common::cmd_getval;
+/*
+  dout priority hierarchy
+  1 - initializations in/out, any critical error
+  2 - initialization results
+  3 - allocation failures
+  5 - init/cleanup logs
+  8 - gifting/reclaiming BlueFS space
```

Comment:
IMO that's an overkill specification - it highly depends on internal details. I'd prefer more general class/priority description  e.g 'external impact in operation' vs. class 1 = 'external impact while initializing (which to replace initialization in/out).
The same applies to class 3 (allocation failures)